src/common/adjacency_ransac.cpp

```diff
@@ -36,14 +36,16 @@
 #include <boost/foreach.hpp>
 #include <boost/shared_ptr.hpp>
 
-#include <opencv2/core/core.hpp>
-#include <opencv2/features2d/features2d.hpp>
-#include <opencv2/highgui/highgui.hpp>
-
 #include "adjacency_ransac.h"
 #include "ransac.h"
 #include "sac_model_registration_graph.h"
 
+#ifndef CV_VERSION_EPOCH
```

Comment:
The latest OpenCV3 should be fine with that file, you don't need to modify it. Thx